### ISSUE TITLE
Invalid disambiguation symbol

### ISSUE THREAD

**Issue by yo35** [yo35/kokopu#2]
```
var kokopu = require('./index');

var p = new kokopu.Position('2r1b2k/p5p1/1p1qp2p/3nNp2/3P3Q/P2R3P/1P3PP1/1Br1R1K1 w - - 0 25');
console.log(p.ascii());
// +---+---+---+---+---+---+---+---+
// |   |   | r |   | b |   |   | k |
// +---+---+---+---+---+---+---+---+
// | p |   |   |   |   |   | p |   |
// +---+---+---+---+---+---+---+---+
// |   | p |   | q | p |   |   | p |
// +---+---+---+---+---+---+---+---+
// |   |   |   | n | N | p |   |   |
// +---+---+---+---+---+---+---+---+
// |   |   |   | P |   |   |   | Q |
// +---+---+---+---+---+---+---+---+
// | P |   |   | R |   |   |   | P |
// +---+---+---+---+---+---+---+---+
// |   | P |   |   |   | P | P |   |
// +---+---+---+---+---+---+---+---+
// |   | B | r |   | R |   | K |   |
// +---+---+---+---+---+---+---+---+
// w - -

var md = p.notation('Rdd1');
console.log(p.notation(md)); // Returns Rd1 instead of Rdd1.
```

**Comment by yo35** [comment]
Fixed in 1.2.1

**Comment by yo35** [comment]
PS : see also yo35/rpb-chessboard#146.
